### DISCUSSION TITLE
Evolution report menu notes column

### DISCUSSION THREAD

**Discussion by TrapinchO** [DraqueT/PolyGlot#1057] [Q&A] [answered]
While playing with the evolution tool, I saw that the report has 3 **read-only** columns. Two for before and after word forms and third for notes. The last one is completely blank though. The user manual also doesn't seem to mention the report.
![img](https://user-images.githubusercontent.com/67415128/143931197-c7b28552-0c12-4b3b-84a3-79d08802ffc7.png)

**Comment by DraqueT** [comment, marked as answer]
The notes column can show a variety of information. If there's some kind of error in the conversion of the word (due to complex regex scenarios most commonly), an explanation of the failure goes there. If there are manually created conjugations (for words with nonstandard forms or something), then their conversions are noted here. If evolution of word transforms themselves has been selected, they will be noted here as well. 

The last is particularly worth noting, as evolving the transformations can be very tricky, especially if you're using heavy regex.

**Reply by TrapinchO** [reply]
Thanks for the answer!
Should I make an issue for the documentation?

**Reply by DraqueT** [reply]
Yeah, that would be good, thanks! That should definitely be added before release.
